app/ui-message/client/messageBox/messageBoxAudioMessage.js

```diff
@@ -95,6 +119,9 @@ Template.messageBoxAudioMessage.events({
 				instance.time.set(`${ String(minutes).padStart(2, '0') }:${ String(seconds).padStart(2, '0') }`);
 			}, 1000));
 			recordingRoomId.set(this.rid);
+			recordingIndicatorInterval.set(setInterval(() => {
+				UserAction.start(this.rid, USER_RECORDING, { tmid: this.tmid });
```

Comment:
Same case I've mentioned for video recording here, if there is an option to start the activity and ignore the timeout this interval is not necessary anymore.